packages/os/load-kernel-modules.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Load additional kernel modules
+RefuseManualStart=true
+RefuseManualStop=true
+After=link-kernel-modules.service
+Wants=link-kernel-modules.service
```

nit: could strengthen this dependency -
```suggestion
Requires=link-kernel-modules.service
```